images/php-fpm/00-lagoon-php.ini.tpl

```diff
@@ -8,6 +8,7 @@ display_startup_errors = ${PHP_DISPLAY_STARTUP_ERRORS:-Off}
 auto_prepend_file = ${PHP_AUTO_PREPEND_FILE:-none}
 auto_append_file = ${PHP_AUTO_APPEND_FILE:-none}
 error_reporting = ${PHP_ERROR_REPORTING:-E_ALL & ~E_DEPRECATED & ~E_STRICT}
+error_log=${PHP_ERROR_LOG:/proc/self/fd/2}
```

Comment:
To use a default value in [parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html), the syntax is `${parameter:-word}`, you're just missing the `-`.

Reply:
Right, tx, I thought I have already forcr pushed a fix, but it looks like I did not. Coming tomorrow...